detections/cloud/cisco_isovalent___potential_escape_to_host.yml

```
@@ -0,0 +1,87 @@
+name: Cisco Isovalent - Potential Escape to Host
+id: 2b8a7a21-bec6-4e1f-84c4-7b319f45d2ab
+version: 1
+date: '2025-11-18'
+author: Bhavin Patel, Splunk
+type: TTP
```

I would put this to anomaly just because the combination of `ps` and `ls` could lead to a high rate of FPs.
```suggestion
type: Anomaly
```